app/src/main/java/org/fossasia/openevent/general/similarEvents/EventTopicsDao.kt

```diff
@@ -0,0 +1,31 @@
+package org.fossasia.openevent.general.similarEvents
+
+import android.arch.persistence.room.Dao
+import android.arch.persistence.room.Insert
+import android.arch.persistence.room.OnConflictStrategy
+import android.arch.persistence.room.Query
+import io.reactivex.Flowable
+import org.fossasia.openevent.general.event.Event
+
+@Dao
+interface EventTopicsDao {
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    fun insertSimilarEvents(events: List<Event>)
+/*
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    fun insertEventTopic(eventTopic: EventTopic)
+*/
+    @Query("DELETE FROM Event")
+    fun deleteAllEvents()
+/*
+    @Query("DELETE FROM EventTopic")
+    fun deleteAllEventTopics()
+*/
```

Comment:
?

Reply:
I thought I might probably require these methods in future, will probably remove them for now :+1:

Reply:
yes please remove this dont keep commented code in final version